File: cash.c
#include <cs50.h>
#include <stdio.h>

int calculate_quarters(int cents);

int main(void)
{
    int cents, quarters;
    do
    {
        cents = get_int("Change owed: ");
    }
    while (cents < 0);
    quarters = calculate_quarters(cents);
    printf("%d\n", quarters);
}
int calculate_quarters(int cents)
{
    int i = 0;
    while (cents >= 25)
    {
        cents -= 25;
        i++;
    }
    while (cents >= 10)
    {
        cents -= 10;
        i++;
    }
    while (cents >= 5)
    {
        cents -= 5;
        i++;
    }
    while (cents > 0)
    {
        cents--;
        i++;
    }
    return i;
}

File: cash.py
from cs50 import get_float
t = get_float("Change: ")
p = 0
while t >= 0.25:
    t -= 0.25
    p += 1
while t >= 0.10:
    t -= 0.10
    p += 1
while t >= 0.05:
    t -= 0.05
    p += 1
while t >= 0.01:
    t -= 0.01
    p += 1
print(f"{p}")
